RELEASE.md

```diff
@@ -1,35 +1,74 @@
+This release enhances the modelling of Eligible Collateral Criteria to enable the use of complex AND, OR and NOT logic in the combination of terms within a criteria.
+
+Eligible Collateral is currently modelled using the data type `EligibleCollateralSpecification` which can contain many `EligibleCollateralCriteria`, which are themselves constructed from `CollateralTreatment`, `IssuerCriteria` and `AssetCriteria`.
+The attributes `isIncluded` (true/false) and `qualifier` (all/any) can be used to model some simple cases of and / or logic in the construction of certain parts of the criteria (eg `AgencyRatingCriteria`).
+
+Members of the CDM Collateral Working Group have requested that the functionality is extended to enable more complex combinations of AND and OR logic across multiple terms.  This has been implemented by combining all the criteria terms in a single new data type `CollateralCriteria`.  This is a choice data type that includes all the criteria terms that previously appeared in `AssetCriteria` and `IssuerCriteria`.  As each term can only occur once, in its simplest form, only one criteria can be specified.  If more than one criteria is required, it is necessary to specify whether the terms are all required or only that any of them are required.  The two new data types `AllCriteria` and `AnyCriteria` also appear on `CollateralCriteria` to enable this.  If all terms are required, ie AND logic, then the terms should be linked in a parent `AllCriteria`.  If any of the terms are required, ie OR logic, then the terms should be linked in a parent `AnyCriteria`.  These two terms can be used iteratively to create complex logic between terms.  Additionally, the data type `NegativeCriteria` can also be used in the logic to apply a NOT function to a single term.
 
 _What is being released?_
 
-This release creates the following modifications:
+This release implements AND, OR and NOT logic between the Collateral terms.
+
+There is no longer a separate data type for each of asset and issuer criteria; they have been combined in a single new data type called `CollateralCriteria`.
+- The new choice data type `CollateralCriteria` replaces the removed `AssetCriteria` and `IssuerCriteria` data types as the combined type.
+- The attributes `issuer` and `asset` on `CollateralCriteriaBase` have now been replaced with the single one `collateralCriteria` which is the specific criteria that applies. It can be created using AND, OR and NOT logic, and both asset and issuer characteristics.
+- The conditions on `CollateralCriteriaBase` have been updated and now use the new `CriteriaMatchesAssetType` function.
+- The data type `ConcentrationLimit` has been refactored to reduce the cardinality of `concentrationLimitCriteria` to 1 and the condition is updated accordingly.
+- The condition on `ConcentrationLimitCriteria` has been updated to reflect the combinaed `CollateralCriteria`.
```

Comment:
combined

Reply:
Rest of the changes look good, just that little typo and then good to go!

Reply:
That typo corrected plus a rogue "#" sign removed